controllers/cloud.redhat.com/makers/maker.go

```diff
@@ -116,8 +116,13 @@ func (m *Maker) makeService(deployment crd.Deployment, app *crd.ClowdApp) error
 		{Name: "metrics", Port: m.Env.Spec.Providers.Metrics.Port, Protocol: "TCP"},
 	}
 
-	if deployment.Web == true {
-		webPort := core.ServicePort{Name: "web", Port: m.Env.Spec.Providers.Web.Port, Protocol: "TCP"}
+	if deployment.Web == true || deployment.WebServices.Public.Enabled == true {
+		webPort := core.ServicePort{Name: "public", Port: m.Env.Spec.Providers.Web.Port, Protocol: "TCP"}
```

Comment:
Changing this won't change anything for current apps - this will change the "name" of the port in the Service itself, unless we use that name somewhere to refer to the service.............

Reply:
oohh yeah, we do: in the ServiceMonitor.

Reply:
After we talked about this some more, I realized that the port name is only used for metrics, not the public/internal ports.  We should be good here.